### DISCUSSION TITLE
How to get the node number?

### DISCUSSION THREAD

**Discussion by YuYuJou** [OpenFAST/openfast#2533] [Q&A] [answered]
I want to output "Total and Additional Loads" and internal forces on mooring lines (L#N@), but I don't know how to get node numbers.
![image](https://github.com/user-attachments/assets/7495f8f6-eb84-4e73-997d-cefd2dd7c5e4)
IF I output con1fx and L1N51fx, are they the same?

Thank you to whoever answer my question.

**Comment by RyanDavies19** [comment, marked as answer]
Hi @YuYuJou, 

MoorDyn has updated from using Con to Point for describing point objects. However, to answer your question, POINT1FX and LINE1N51FX should be very similar. There might be minor differences depending on how / if you have point hydrodynamics. The motion of Point 1 and Line1N51 will be identical.  The MoorDyn documentation has some more information about available output channels: https://moordyn.readthedocs.io/en/latest/inputs.html#id5

If you want the internal forces at every node along the line, you can add flags to the outputs column of the lines list: https://moordyn.readthedocs.io/en/latest/inputs.html#id3

**Comment by YuYuJou** [comment]
I output "L3N50pz" and it worked.
But I tried to output "L3N50fz", it said "unsupported".
I want to know why I couldn't output the internal force, is it because of time long, the external force, or something?

**Reply by RyanDavies19** [reply]
Hi @YuYuJou, I don't see that same error when I try to replicate things on my end. L3N50fz is a valid output string. A couple points to check: Are you using the most recent release of OpenFAST? What is the exact error that MoorDyn produces? What does your entire MoorDyn input file look like when the error is shown?

**Comment by YuYuJou** [comment]
![image](https://github.com/user-attachments/assets/e38b4cc9-4368-4ed7-abb1-4f55d45614c7)
I tried different points to output the internal force, and all showed me that.
I use v3.3.0. 
I will try the latest version.
Thank you.

**Reply by RyanDavies19** [reply]
Ah, that makes sense. There have been a number of improvements to MoorDyn since v3.3.0, and the MoorDyn docs reflect the current MoorDyn features in the most recent release (or sometimes the dev branch). If you want to get the segment tensions, you should be able to in v3.3.0 by proving the t flag in the line list output flags.

**Reply by YuYuJou** [reply]
Sorry, I have 1 more question.
![image](https://github.com/user-attachments/assets/be55b659-586e-4627-852d-bab0d4dc00a6)
Because you said "MoorDyn has updated from using Con to Point for describing point objects.", and that's the result that I outputted 'Con5fz', 'L3N50fz' & 'Point5fz'.
 'Con5fz'&'L3N50fz' are the same, but 'Point5fz' is quite different, I want to know why.

**Reply by RyanDavies19** [reply]
Hi @YuYuJou, 

Are you able to provide a screenshot or text of your whole input file here? This should not be the case as Point# and Con# are equivalent keys for the same output channel in the most recent version of MoorDyn.

**Comment by YuYuJou** [comment]
![image](https://github.com/user-attachments/assets/79a6ade1-715e-44eb-8b83-d8b3af93318e)
I use v3.5.3

**Reply by RyanDavies19** [reply]
@YuYuJou I am unable to replicate your error using OpenFAST v3.5.3. All of the following channels give identical results for me:
```
L1N50fz
L1NBfz
Point1fz
Con1fz
```

**Reply by RyanDavies19** [reply]
@YuYuJou, have you been able to come up with a solution?

**Comment by YuYuJou** [comment]
Sorry, I downloaded the file and reran the same work. 
Point1fz & Con1fz gave the same result.
I might change something without notice.
Thanks for your help.
